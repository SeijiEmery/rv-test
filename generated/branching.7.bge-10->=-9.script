load /x 0 generated/branching.7.bge-10->=-9.hex
writereg 10 10
writereg 11 9
writereg 6 1
run 0 100000
readreg 10
readreg 11
readreg 6