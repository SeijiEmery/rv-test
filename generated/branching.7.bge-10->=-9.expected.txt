R10 = 10
R11 = 9
R6 = 1
